ast/annotations.go

```diff
@@ -261,8 +308,13 @@ func (a *AuthorAnnotation) Compare(other *AuthorAnnotation) int {
 }
 
 func (a *AuthorAnnotation) String() string {
-	bs, _ := json.Marshal(a)
-	return string(bs)
+	if len(a.Email) == 0 {
+		return a.Name
+	} else if len(a.Name) == 0 {
+		return fmt.Sprintf("<%s>", a.Email)
+	} else {
+		return fmt.Sprintf("%s <%s>", a.Name, a.Email)
+	}
```

Comment:
[style] I'd have chosen a condition-less switch,

```suggestion
	switch {
	case len(a.Email) == 0:
		return a.Name
	case len(a.Name) == 0:
		return fmt.Sprintf("<%s>", a.Email)
	default:
		return fmt.Sprintf("%s <%s>", a.Name, a.Email)
	}
```

but that's merely a matter of personal preference.